src/pages/journal/journal.html

```diff
@@ -8,13 +8,24 @@
 </ion-header>
 
 <ion-content padding>
+  <ion-refresher (ionRefresh)="pullRefreshJournal($event)">
```

Comment:
Having tested on the iPad, the distance pulled down to trigger the refresh seems quite small. Could you perhaps set `pullMin` and `pullMax` to make it a bit more ergonomic?

Reply:
Yes we are now clearing out the previously dynamically generated slots

Reply:
Yep can have a look at increasing the pull